src/components/Projects/ProjectGeneralInfo.tsx

```diff
@@ -15,15 +15,15 @@ function ProjectGeneralInfo({ project }: Props) {
   const t = useFormatMessage()
 
   return (
-    <div className="ProjectSidebar__ContentContainer">
+    <>
```

Comment:
Is this ok?

Reply:
Yes! The container it's outside the component